examples/src/test/resources/transfer.properties

```diff
@@ -3,8 +3,9 @@ node.manager.address=https://testnet-nodemanager.coti.io
 full.node.backend.address=https://testnet-fullnode1.coti.io
 trust.score.backend.address=https://testnet-trustscore1.coti.io
 seed=
-source.address.index=0
+source.address.index=1
+target.address.index=2
```

Comment:
add a comment above the line that explans what this controls. name is not self explanatory